corehq/apps/registration/static/registration/js/password.js

```diff
@@ -0,0 +1,30 @@
+var PasswordModel = function (weak, better, strong) {
+    var self = this;
+    self.weakMessage = weak;
+    self.betterMessage = better;
+    self.strongMessage = strong;
+    self.password = ko.observable('');
+    self.strength = ko.computed(function() {
+        return zxcvbn(self.password(), ['dimagi', 'commcare', 'hq']).score
```

Comment:
Can you split out this array into a variable so that its clear what it does? e.g. `penalized_passwords = [...]`